pkg/kube/prowjob.go

```diff
@@ -69,6 +69,9 @@ const (
 	// IsOptionalLabel is added in resources created by prow and
 	// carries the Optional from a Presubmit job.
 	IsOptionalLabel = "prow.k8s.io/is-optional"
+	// ReRunLabel is added in periodics that are configured to be
+	// re-runned several times
```

Comment:
"configured to be-rerunned" does not seem to be correct. IIUC the jobs which _are retries_ will have this label?

Ideally also document the expected value (position in the series of retries) or something like this.